xbmc/interfaces/legacy/ListItem.h

```diff
@@ -998,6 +999,22 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
       ///
       /// \ingroup python_xbmcgui_listitem
+      /// @brief \python_func{ getPictureInfoTag() }
+      ///-----------------------------------------------------------------------
+      /// Returns the PictureInfoTag for this item.
+      ///
+      /// @return     picture info tag
+      ///-----------------------------------------------------------------------
+      /// @python_v17 New method added.
```

Comment:
v18 and some other places as well ;) 
Need to add this to some file to make  it visible for doxygen